src/components/Preferences/Import.js

```
@@ -0,0 +1,83 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { useDropzone } from 'react-dropzone'
+import { Button } from '@aragon/ui'
+import { isString } from '../../utils'
+import { isAddress } from '../../web3-utils'
+
+// What is the answer to the ultimate question of Life, the Universe, and Everything?
+const MAX_LENGTH = 42
+
+const verifyLocalIdentityObject = obj => {
+  return (
+    Array.isArray(obj) &&
+    obj.every(
+      ({ address, name, createdAt }) =>
+        !!address.trim() &&
+        !!name.trim() &&
+        !!createdAt &&
+        isAddress(address) &&
+        isString(name) &&
+        name.length <= MAX_LENGTH
+    )
+  )
+}
+
+const fileImport = cb => files => {
+  if (!files || !files.length) {
```

Should we also throw an error here if there's more than one file?